test/intl402/Temporal/Instant/formatrange-fails.js

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.instant
+description: formatRange fails if given arguments of different types
+features: [Temporal]
+---*/
+
+const us = new Intl.DateTimeFormat('en-US');
+
+const instances = {
+  instant: new Temporal.Instant(0n),
```

Comment:
Oh, I'd also add something like `number: Date.UTC(2000, 5, 2)` to test the combination of a Temporal object with any other value.

Reply:
good idea -- done